tests/boot/test_mcuboot/sysbuild.cmake

```diff
@@ -0,0 +1,30 @@
+# Copyright 2022 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+# Add the mcuboot key file to the secondary swapped app
+
+if(SB_CONFIG_BOOTLOADER_MCUBOOT)
```

Comment:
Why is the `if` necessary, if the point of the sample is to test this?

Reply:
It's not- I've removed this